### ISSUE TITLE
how to set default maker for specific language?

### ISSUE THREAD

**Issue by jidibinlin** [neomake/neomake#2433]
I want to set the clangcheck as the maker for c/cpp. So how to make it real?

**Comment by blueyed** [comment]
```
let g:neomake_c_enabled_makers = ['cppcheck']
let g:neomake_cpp_enabled_makers = ['cppcheck']
```

I think having better defaults for C/C++ would be good, since it often needs additional setup.
I've some WIP somewhere to enable the clang based makers also in this regard.

**Comment by jidibinlin** [comment]
Thank you! It`s work for me. And use clangcheck can simple fix this issue #1925 for what i think when we use cmake to generate a compile_command.json.
